File: JavaScript/1593.js
// Problema: 1593 - Função Binária   | Resposta: Accepted
// Linguagem: JavaScript 12.18 [+2s] | Tempo: 0.047s

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.trim().split('\n');

lines.shift();
lines.forEach((line)=> {
    binario = BigInt(line).toString(2);
    console.log(binario.split('1').length - 1);
});

File: Python/1593.py
# Problema: 1593 - Função Binária | Resposta: Accepted
# Linguagem: Python 3.11 [+1s]    | Tempo: 0.000s

for _ in range(int(input())):
    dec = int(input())
    number_bin = bin(dec)[2:]
    str_bin = str(number_bin)
    print(str_bin.count('1'))
